docs/Learning-Environment-Design-Agents.md

```diff
@@ -507,6 +509,49 @@ setting the State Size.
 - Use as few rays and tags as necessary to solve the problem in order to improve
   learning stability and agent performance.
 
+### Variable Length Observations
+
+It is possible to have observations be of variable size by using a `BufferSensor`,
```

Comment:
```suggestion
It is possible for agents to collect observations from a varying number of GameObjects by using a `BufferSensor`.
```

Reply:
Suggestion on the suggestion - I don't think mentioning "GameObjects" is neecssary